lib/fx/function.rb

```diff
@@ -3,23 +3,26 @@ module Fx
   class Function
     include Comparable
 
-    attr_reader :name, :definition
+    attr_reader :name, :arguments, :definition
     delegate :<=>, to: :name
 
-    def initialize(function_row)
-      @name = function_row.fetch("name")
-      @definition = function_row.fetch("definition")
+    def initialize(function)
+      @name = function.fetch("name")
+      @arguments = function.fetch("arguments", [])
+      @definition = function.fetch("definition")
     end
 
     def ==(other)
-      name == other.name && definition == other.definition
+      name == other.name &&
+        definition == other.definition &&
+        arguments == other.arguments
     end
 
     def to_schema
-      <<-SCHEMA.indent(2)
```

Comment:
Yup! I actually noticed indentation issues when using this in our Rails project.